specification/storage/resource-manager/Microsoft.Storage/stable/2021-01-01/blob.json

```diff
@@ -815,6 +815,46 @@
           }
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Storage/storageAccounts/{accountName}/blobServices/default/containers/{containerName}/migrate": {
+      "put": {
+        "tags": [
+          "BlobContainers"
+        ],
+        "operationId": "BlobContainers_VersionLevelWorm",
+        "x-ms-examples": {
+          "VersionLevelWormContainerMigration": {
+            "$ref": "./examples/VersionLevelWormContainerMigration.json"
+          }
+        },
+        "description": "This operation migrates a blob container from container level WORM to version level WORM enabled container. Prerequisites require a container level immutability policy either in locked or unlocked state, Account level versioning must be enabled and there should be no Legal holds.",
+        "parameters": [
+          {
+            "$ref": "./storage.json#/parameters/ResourceGroupName"
+          },
+          {
+            "$ref": "./storage.json#/parameters/StorageAccountName"
+          },
+          {
+            "$ref": "#/parameters/ContainerName"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK -- returned when the blob container migration to support version level WORM completes."
+          },
+          "202": {
+            "description": "Accepted -- Blob Container migration to support version leve worm is accepted; operation will complete asynchronously."
+          }
+        },
+        "x-ms-long-running-operation": true
```

Comment:
please specify final-state0via according to https://github.com/Azure/autorest/blob/master/docs/extensions/readme.md#x-ms-long-running-operation-options

Reply:
azure-async-operation - (default if not specified) poll until terminal state, the final response will be available at the uri pointed to by the header Azure-AsyncOperation

By default it will use the above, which I need so will keep it like this if no worries